### PR TITLE
[entropy complex/security] countermeasure lists and labels updated

### PR DESCRIPTION
**Pull request by mwbranstad**

### CODE REVIEW

File: hw/ip/entropy_src/data/entropy_src.hjson
Comment (cdgori) [inline hw/ip/entropy_src/data/entropy_src.hjson:98]: nit: generater -> generator

File: hw/ip/csrng/data/csrng.hjson
Comment (cdgori) [inline hw/ip/csrng/data/csrng.hjson:101]: INTEGRITY -> CONSISTENCY for these consecutive values checks

File: hw/ip/edn/data/edn.hjson
Comment (cdgori) [inline hw/ip/edn/data/edn.hjson:66]: INTEGRITY -> CONSISTENCY for these consecutive values checks

File: hw/ip/entropy_src/data/entropy_src.hjson
Comment (cdgori) [inline hw/ip/entropy_src/data/entropy_src.hjson:103]: INTEGRITY -> CONSISTENCY for these consecutive values checks

**Comment by cdgori** [comment]
Please adjust per comments above.  You'll also need to add another reviewer with commit rights or this will be stuck.  It won't even let me start a "review" because of the missing committer.

**Comment by tjaychen** [comment]
this looks good to me.

**Review by tjaychen — APPROVED** [review]

**Pull request merged**
